judo/gui.py

```diff
@@ -245,15 +264,39 @@ def _get_callback(
 
     def gui_callback(_: GuiEvent) -> None:
         """Updates the config with the new value received from viser."""
-        if isinstance(element, GuiSliderHandle):
+        if isinstance(element, GuiBoundedSliderHandle):
```

Comment:
Why do we need a separate class? Isn't this always undesirable behavior? Shouldn't we just always perform the checks you're doing?